### DISCUSSION TITLE
Prevent Tailwind colors to be changed by Tailwind UI

### DISCUSSION THREAD

**Discussion by MartijnHarmenzon** [tailwindlabs/tailwindcss#1776] [Help] [answered]
When adding a modal, using the example of [Tailwind UI](https://tailwindui.com/components/application-ui/overlays/modals), all my default Tailwind colors changed. When including the @tailwindcss/ui plugin, the Tailwind color pallete probably changed.
Is anyone else experiencing the same problem? And how to resolve?

**Comment by Apostolique** [comment, marked as answer]
You can see the new color palette here: https://tailwindui.com/documentation#how-tailwindcss-ui-extends-tailwind

**Reply by MartijnHarmenzon** [reply]
Thanks! That explains why.
